file.go

```diff
@@ -133,7 +133,15 @@ func (p *FileProvider) Read(sid string) (_ RawStore, err error) {
 	defer p.lock.RUnlock()
 
 	var f *os.File
+	ok := false
```

Comment:
I think rename this variable to be `expired` is better.

Reply:
That would require the following changes:

```suggestion
	expired := false
```